jui-grid/index.d.ts

```diff
@@ -0,0 +1,996 @@
+// Type definitions for jui-grid 2.0
+// Project: https://github.com/juijs/jui-grid
+// Definitions by: Jin-Ho Park <https://github.com/easylogic>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+
+import {UIEvent} from 'jui-core';
+
+export interface GridColumn {
+    (index: number|string): this;
```

Comment:
Thanks for your comment.

The JUI library class is internally different from the normal class, so it is more appropriate to define it as an interface.